### DISCUSSION TITLE
Multiple writing processes

### DISCUSSION THREAD

**Discussion by sergey-miryanov** [hse-project/hse#561] [Q&A] [answered]
Hi!
I saw in the documentation that hse supports thousands of concurrent operations and this is sounds good! Can I hse from multiple processes to write data to the one database?

**Comment by tristan957** [comment, marked as answer]
@sergey-miryanov you cannot. Everything HSE does is within a single process.

**Reply by sergey-miryanov** [reply]
Thank you for your reply!

**Reply by tristan957** [reply]
No problem. Please let us know if any other questions or issues arise :)
